tests/quantization/test_quantized_ops.py

```diff
@@ -363,8 +363,9 @@ def test_all_arith_ops(
         # Compute the quantized operator result
         quantized_output_vv = q_op(q_inputs_0, q_inputs_1).dequant()
 
-        # Check the R2 of raw output and quantized output
-        check_r2_score(raw_output_vv, quantized_output_vv)
+        if n_bits > 16:
```

Comment:
are we sure about this change ?

Reply:
not sure I see where we were doing the second chck 🤔

Reply:
`        check_r2_score(raw_output_vv, quantized_output_vv)` is right below the if

Reply:
Ah yeah what am saying -_-'. Sorry. So yes the problem is as I said, correctness were fine before with univariate LUT but in QuantizedDiv|Mul we have quant requant which impact the final accuracy.

Reply:
do we know by how much it can impact ? how "strong" do we expect this impact to be ? I'm just being a bit suspicious here because of things like https://github.com/zama-ai/concrete-ml/pull/690/files#r1634728461 or https://github.com/zama-ai/concrete-ml/pull/690/files#r1634723604 😅